### PR TITLE
[Merged by Bors] - fix: changelog.md

### PR DESCRIPTION
**Pull request by XtremeDevX**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Release Notes
 
 ## Platform Version 0.9.24 - UNRELEASED
-* CLI: Migrate all fluvio crates to `comfy-table` from `prettytable-rs` ([#2285](https://github.com/infinyon/fluvio/pulls/2285))
+* CLI: Migrate all fluvio crates to `comfy-table` from `prettytable-rs` ([#2285](https://github.com/infinyon/fluvio/pull/2285))
```
Comment (morenol) [inline CHANGELOG.md:4]: Could you point to the issue instead?

**Review by morenol — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (XtremeDevX) [reply CHANGELOG.md:4]: Sorry, what do you mean?

Comment (morenol) [reply CHANGELOG.md:4]: This is a link to the previous PR. We should add a link to the issue #2263 instead

**Review by morenol — APPROVED** [review]
LGTM

**Pull request closed without merging**